### ISSUE TITLE
Support importing movies from ...

### ISSUE THREAD

**Issue by StuartJAtkinson** [NoelDeMartin/media-kraken#16]
Simklr like trackt they have a pretty good api I think

**Comment by NoelDeMartin** [comment]
Hey, thanks for opening your request.

I just created an account in [simkl.com](https://simkl.com) (I suppose that's what you meant), and I can't find a way to export my data. The only option I found is [the backup app](https://simkl.com/apps/backup/); but it requires a VIP/PRO account which I don't have. Is that what you meant? If that's it, I need an example file to see the format (both with watched movies & movies to watch later).

There is also a free option to export data using the API, but I don't think that's user-friendly, and it's also harder to implement because Media Kraken only supports importing from a file at the moment. So I'm not sure if I'd bother adding support to this. If someone's a developer that would set up the developer API, they can already use it themselves to generate a jsonld file which Media Kraken already supports.
